Source/HelixToolkit.Maths/Matrix4x4Extensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HelixToolkit.Maths
+{
+    public static class Matrix4x4Extensions
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static Matrix4x4 Inverted(this Matrix4x4 m)
+        {
+            return Matrix4x4.Invert(m, out var result) ? result : Matrix4x4.Identity;
```

Comment:
We probably don't want a silent failure if the matrix is not able to be inverted.

Reply:
Return identity matrix is still a bug. We can log the error if it's not able to be inverted.

Reply:
You can return either identity or zeros, just log the error.